modules/client/src/client-common/modules/graphql.ts

```diff
@@ -76,14 +66,40 @@ export class GraphqlModule implements IClientGraphQLCore {
    */
   public async ensureOnline(): Promise<void> {
     if (!this.clients.length) {
-      return Promise.reject(new Error("graphql client is not initialized"));
+      throw new ClientNotInitializedError("graphql");
     }
-
     for (let i = 0; i < this.clients.length; i++) {
       if (await this.isUp()) return;
-
       this.shiftClient();
     }
-    throw new Error("No graphql nodes available");
+    throw new NoNodesAvailableError("graphql");
+  }
+
+  public async request({ query, params, name }: {
```

Comment:
Async is not needed with only one `return`

```suggestion
  public request({ query, params, name }: {
```